errors.go

```diff
@@ -25,387 +25,99 @@ package errdefs
 
 import (
 	"context"
-	"errors"
+	"fmt"
 )
 
+type Error int8
```

Comment:
Can you rename this to `errNum` or `ErrNum` if a const must be an exported type.